xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -392,6 +392,22 @@
     },
     "additionalProperties": { "type": [ "null", { "$ref": "Global.String.NotEmpty", "required": true } ] }
   },
+  "Video.Rating": {
+    "type": "object",
+    "properties": {
+      "rating": { "type": "number", "required": true },
+      "votes": { "type": "integer" },
+      "default": { "type": "boolean" }
+    }
+  },
+  "Video.Ratings": {
+    "type": "object",
+    "additionalProperties": { "type": "$ref": "Video.Rating" }
+  },
+  "Video.Ratings.Set": {
+    "type": "object",
+    "additionalProperties": { "type": [ "null", { "$ref": "Video.Rating", "required": true } ] }
```

Comment:
That's because the "Video.Ratings" was already there before this change. I just moved and changed the definition. So all the times it was used (except 3 I changed now to Video.Ratings.Set) are still there. You just can't see it from the diff ;)
Look at the full file and you will find them
https://github.com/phate89/xbmc/blob/4b9fb67521205adf2c48a675d13db0f8e96200e5/xbmc/interfaces/json-rpc/schema/types.json